src/DotVVM.Framework/Resources/Scripts/serialization/date.ts

```diff
@@ -1,14 +1,30 @@
 import { logWarning } from "../utils/logging";
 
+const timeSpanBaseDate = new Date("0001-01-01 00:00:00.0000");
```

Comment:
I don't understand what is going on, but I'm afraid this may break in some cases:
![image](https://user-images.githubusercontent.com/7894687/115057549-b8c2e300-9ed3-11eb-9918-dd79ee1020d9.png)

Reply:
Aha, the year 0001 is parsed in Node in non-UTC timezone as year 2001 :facepalm: Do we have this covered with (unit) tests?